entrypoint.sh

```diff
@@ -115,6 +118,10 @@ fi
 if [ "$hideProgress" == "true" ];then
   ARGS="$ARGS --no-progress"
 fi
+
+if [ "$listAllPkgs" == " true" ];then
```

Comment:
NOTE: white space is required before `true` because entrypoint.sh receives option as `"-r true"`, not `-r true`. An argument of `"-r true"` option will be ` true` (with whitespace prefix). Then this code works.

See an example of [testing the action](https://github.com/m-mizutani/trivy-opa-example/runs/4490956015?check_suite_focus=true) at 5th step "Run Trivy vulnerability scanner in repo mode"

Reply:
You can trim \r. Check examples in this file.

Reply:
@simar7 Thank you for your comment. I fixed to trim whitespace at b642dfb